src/pkg/caendr/caendr/services/sql/etl/phenotype_metadata.py

```diff
@@ -0,0 +1,70 @@
+import csv
+
+from caendr.services.logger import logger
+
+from caendr.models.datastore  import Species
+from caendr.utils.local_files import LocalDatastoreFile
+
+
+def parse_phenotype_metadata(species: Species, **files: LocalDatastoreFile):
+  """
+    Parsing function to load PhenotypeMetadata table from datastore.
+    For bulk file ouputs each trait as a new row and replicates metadata for all of them
+  """
+  
+  logger.info('Parsing function for traits metadata')
+  
+  for file_name, file_object in files.items():
+
+    # Get metadata for the file 
+    md = file_object.metadata
+
+    # If bulk file, open it and yield each trait as a new row with the file's metadata
+    if md.is_bulk_file:
+      with open(file_object) as csv_file:
+        for idx, row in enumerate( csv.reader(csv_file, delimiter='\t') ):
```

Comment:
it might be useful to document here the expected order of items in the `row`. 

Also, consider unpacking the row into named local vars to ensure we have the correct expectations, eg: 
```
row = [1,2]
foo, bar = row 
```

then use the named variables to replace places where we have `row[1]`.